src/components/Beer.vue

```diff
@@ -44,6 +55,11 @@ export default {
   props: {
     beer: Object
   },
+  methods: {
+    isBeerFavourite() {
+      return (this.beer.isFavourite = !this.beer.isFavourite);
```

Comment:
should this not be `===`? Are you assigning a new value on purpose?

If you are just returning a boolean, without using arguments, a computed property is more performant

Reply:
This method is toggling the state of the beer object property isFavourite.

This is then used to add a conditional class ```:class="{ 'beer--favourite': beer.isFavourite }"```

And use in the computed property for the button text

```
    beerFavouriteText() {
      return this.beer.isFavourite
        ? 'Remove from favourites'
        : 'Add to favourites';
    }
```